CHANGELOG.md

```diff
@@ -50,6 +50,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1869](https://github.com/realm/SwiftLint/issues/1869)
 
+* Add `missing_docs` rule to warn against undocumented declarations.  
+  [Nef10](https://github.com/Nef10)
+  [Andrés Cecilia Luque](https://github.com/acecilia)
+  [#1652](https://github.com/realm/SwiftLint/issues/1652)
 * Add a new `ignores_interpolated_strings` config parameter to the `line_length`
```

Comment:
It could have a blank line line to separate the entries. The rest of the file has blank line between consecutive notes.